src/Elcodi/Admin/CoreBundle/Controller/Abstracts/AbstractAdminController.php

```diff
@@ -163,23 +166,24 @@ protected function getResponse(
      *
      * This method takes into account the type of the request ( GET or POST )
      *
-     * @param Request $request     Request
-     * @param string  $redirectUrl Redirect url
+     * @param Request $request      Request
+     * @param string  $redirectPath Redirect path
      *
      * @return mixed Response
      */
-    protected function getOkResponse(Request $request, $redirectUrl = null)
-    {
-        if (null === $redirectUrl) {
-            $redirectRoute = $request->headers->get('referer');
-        } elseif ($redirectUrl instanceof Closure) {
```

Comment:
This was introduced in #47 to generate more elaborate urls, like the one in https://github.com/elcodi/bamboo-admin/blob/933ecbc80938f9477fd045104ba59e9a77f1b58a/src/Elcodi/AdminProductBundle/Controller/VariantController.php#L532-L540. I think this is deprecated by now.